Commands/InteractionCommands/InsidersInteractions.cs

```diff
@@ -0,0 +1,28 @@
+namespace Cliptok.Commands.InteractionCommands
+{
+    public class InsidersInteractions : ApplicationCommandModule
+    {
+        [SlashCommand("send-insiders-info-buttons", "Sends a message with buttons to get Insider roles for #insiders-info.", false)]
+        [SlashRequireHomeserverPerm(ServerPermLevel.Admin, ownerOverride: true), SlashCommandPermissions(permissions: DiscordPermission.ModerateMembers)]
+        public static async Task SendInsidersInfoButtonMessage(InteractionContext ctx)
+        {
+            if (Program.cfgjson.InsiderInfoChannel != 0 && ctx.Channel.Id != Program.cfgjson.InsiderInfoChannel)
```

Comment:
This should also check that `insidersChannel` is specified in the config, to avoid `<#0>` being sent which is difficult to debug

Reply:
Done in https://github.com/Erisa/Cliptok/pull/255/commits/4d349942bb8862f0faef533dc71c50924dce456f; a hardcoded string "#insiders" is sent if the config value is missing (and a warning is logged) because I don't feel the command should fail just for that